public/app/features/dashboard-scene/serialization/transformSceneToSaveModel.ts

```diff
@@ -194,7 +194,12 @@ export function gridItemToPanel(gridItem: SceneGridItemLike, isSnapshot = false)
   }
 
   if (gridItem instanceof PanelRepeaterGridItem) {
-    vizPanel = gridItem.state.source;
+    if (gridItem.state.source instanceof LibraryVizPanel) {
+      vizPanel = gridItem.state.source.state.panel;
+    } else {
+      vizPanel = gridItem.state.source;
```

Comment:
@kaydelaney this can't be correct, with this change we end up with all the library panel json in the dashboard json

Reply:
Ah, will take a look